.github/renovate.json

```diff
@@ -23,6 +23,13 @@
         "patch"
       ],
       "enabled": false
+    },
+    {
+      "matchManagers": [
+        "dockerfile"
+      ],
+      "packageNames": [ "ubuntu" ],
+      "versioning": "regex:^(?<compatibility>[a-z]+)-(?<patch>\\d+)$"
```

Comment:
in case of `ubuntu:jammy-20230624` this should treat the `jammy` term as a constant `compatibility` part and treat the `20230624` with patch semantics.